packages/projects-docs/pages/learn/plans/workspace.mdx

```diff
@@ -0,0 +1,109 @@
+
+![My contribution branches](../images/workspace-my-contributions.png)
+
+### Personal Pro
+<Callout emoji="⚠️">
+Personal Pro has been depricated in favor of [Pro workspaces](##pro-workspace). Existing personal Pro subscriptions will continue to be supported in their current form.  
+</Callout>
+
+
+## Pro Workspace 
+
+Pro workspaces are for private development and collaboration. 
+If you want to work together with a friend or colleague on a sandbox or repository, start by creating a Pro workspace and inviting them to join you.
+
+**Sandboxes**: Anyone with edit permi to the Pro workspace can edit its sandboxes. If you want to collaborate live with workspace members, you can start a [live session](/learn/getting-started/collaborate-share) (with a Pro subscription).
+
+**Repositories**: Similarly, anyone with edit access to the team workspace can edit its repositories — as long as they also have write access to the repository on the git provider (such as GitHub). Branches have their own development environments for each workspace, so files saved on `my-branch` of a repository in your personal workspace will not appear on `my-branch` in the team workspace until they are committed and pushed to the git provider.
+
+### Creating a Pro workspace
+
+You can get started with Pro with a free trial. 
+
+![Create a team](../images/workspace-create-team.png)
+
+Or create a team from the workspace selector:
+
+![Create a team](../images/workspace-selector-newteam.png)
+
+
+### Inviting People to a workspace
+
+You can add as many members to a Pro workspace as you like. You can invite them when you are creating the workspace for the first time, or later through the subscription management view.
+
+![Invite team members](../images/workspace-invite-members.png)
+
+Users with an active CodeSandbox account will get a notification (the bell icon in the header) in CodeSandbox as well as an email. 
+From there, they can accept or decline the invitation.
+
+You can also invite workspace members that do not have a CodeSandbox account. They will be sent an email inviting them to create an account where they will have access to the workspace.
+
+
+### Managing workspaces and subscriptions
+
+You can create and manage workspaces from the dashboard settings page.
+To find this, select your workspace and open the subscription page.
+
+![Team settings](../images/workspace-team-settings.png)
+
+Here you can see the team overview with subscription details, team members and roles.
+Admins can edit team member access and roles here as well as edit the subscription. 
+
+![Team settings](../images/workspace-team-settings-free.png)
+
+
+**Free Team**
+
+<Callout emoji="⚠️">
+Free teams have been depricated. Existing free team subscriptions will continue to be supported in their current form.  
+</Callout>
+
+### Collaboration in workspaces
+
+#### Sharing Sandboxes
+
+You can choose which workspace you're in from the team drop-down at the top-left on
```

Comment:
this paragraph feels a bit long and hard to follow. also not sure how it related to sharing sandboxes, vs sandbox management?